modules/rosa-nodes-machine-pools-local-zones.adoc

```diff
@@ -0,0 +1,76 @@
+* The ROSA cluster was initially built in an existing AWS VPC (BYO-VPC).
+* The maximum transmission unit (MTU) for the ROSA cluster is set at 1200.
++
+[IMPORTANT]
+====
+Generally, the Maximum Transmission Unit (MTU) between an Amazon EC2 instance in a Local Zone and an Amazon EC2 instance in the Region is 1300. See link:https://docs.aws.amazon.com/local-zones/latest/ug/how-local-zones-work.html[How Local Zones work] in the AWS documentation. 
+The cluster network MTU must be always less than the EC2 MTU to account for the overhead. The specific overhead is determined by your network plugin, for example:
+
+- OVN-Kubernetes: `100 bytes`
+- OpenShift SDN: `50 bytes`
+
+The network plugin could provide additional features, like IPsec, that also must be decreased the MTU. Check the documentation for additional information.
+
+====
+* The AWS account has link:https://docs.aws.amazon.com/local-zones/latest/ug/getting-started.html#getting-started-find-local-zone[Local Zones enabled].
+* The AWS account has a link:https://docs.aws.amazon.com/local-zones/latest/ug/getting-started.html#getting-started-create-local-zone-subnet[Local Zone subnet] for the same VPC as the cluster.
+
+.Procedure
+
+. Create a machine pool on the cluster in ROSA CLI.
++
+[source,terminal]
+----
+$ rosa create machinepool -c <cluster-name> -i 
+----
++
+. Add the subnet and instance type for the machine pool in ROSA CLI. After several minutes, the cluster will provision the nodes.
++
+[source, terminal]
+----
+I: Enabling interactive mode <1>
+? Machine pool name: xx-lz-xx <2>
+? Create multi-AZ machine pool: No <3>
+? Select subnet for a single AZ machine pool (optional): Yes <4>
+? Subnet ID: subnet-<a> (region-info) <5>
+? Enable autoscaling (optional): No <6>
+? Replicas: 2 <7>
+I: Fetching instance types <8>
+----
++
+
+<1> Enables interactive mode.
+<2> Names the machine pool. This is limited to alphanumeric and a maximum length of 30 characters.
+<3> Set this option to no.
+<4> Set this option to yes.
+<5> Selects a subnet ID from the list. Associate the subnet with a routing table that has a route to a NAT gateway.
+<6> Set this option to no.
+<7> Selects the number of machines for the machine pool. This number can be anywhere from 1 - 180.
+<8> Selects an instance type from the list. Only instance types that are supported in the selected Local Zone will appear.
++
+. Add the `kubernetes.io/cluster/<infra_id>: shared` tag to the subnet.
+. Provide the subnet ID to provision the machine pool in the Local Zone.
+
+See the link:https://aws.amazon.com/about-aws/global-infrastructure/localzones/locations/[AWS Local Zones locations] list on AWS for generally available and announced AWS Local Zone locations.
+
+[NOTE]
+====
+Local machine pools can be added to single Availability Zone clusters.
```

Comment:
It allows creating a single zone machine pool with subnet(created on local zone) to a multi-az cluster
So the note is not right

Reply:
I've gotten conflicting information on the single zone / multi-az issue. @oriAdler @arendej , can y'all please weigh in?